tests/utils/pod/pod.go

```diff
@@ -0,0 +1,22 @@
+package pod
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+)
+
+func DefinePod(podName string, namespace string, image string, label map[string]string) *corev1.Pod {
```

Comment:
Split this function in to several functions.
DefinePod(podName string, namespace string, image string)
RedefinePodWithLabel(podDefenition *corev1.Pod, label map[string]string)